src/pkg/storage/storage.go

```diff
@@ -8,12 +8,13 @@ import (
 	"github.com/alcionai/corso/src/internal/common"
 )
 
-type storageProvider int
+type Provider int
 
-//go:generate stringer -type=storageProvider -linecomment
+//go:generate stringer -type=Provider -linecomment
 const (
-	ProviderUnknown storageProvider = 0 // Unknown Provider
-	ProviderS3      storageProvider = 1 // S3
+	ProviderUnknown Provider = 0 // Unknown Provider
+	ProviderS3      Provider = 1 // S3
+	ProviderFS      Provider = 2 // FS
```

Comment:
FS is a bit difficult to parse if you don't already know what it means.  Can we use the phrase `LocalFileStorage` instead?  Clarity is preferred over brevity for not-often referenced consts like this.

Reply:
`LocalFileStorage` is misleading too. Since it can be network attached as well. Maybe `Filesystem` would work better.